node/core/dispute-coordinator/src/error.rs

```diff
@@ -123,34 +98,34 @@ pub enum NonFatal {
 
 	/// Errors coming from runtime::Runtime.
 	#[error("Error while accessing runtime information: {0}")]
-	Runtime(#[from] runtime::NonFatal),
+	Runtime2(#[from] runtime::JfyiError),
```

Comment:
`Runtime2`? I think `FatalRuntime` for the former would be more descriptive.

Reply:
Utlimately this will be handled by `#[fatal(defer)]`, so only one variant is needed, but that feature will be left for another PR.

Reply:
Fixed in https://github.com/paritytech/polkadot/pull/4448/commits/6a8edf8999a8267e2a408476a57c1cd28d3e3c45